tests/perfalert/test_alert_modification.py

```diff
@@ -28,6 +53,76 @@ def test_summary_modification(test_repository, test_perf_signature,
     assert s.status == PerformanceAlertSummary.UNTRIAGED
 
 
+def test_summary_status(test_repository, test_perf_signature,
+                        test_perf_alert_summary, test_perf_framework):
+    option_1 = Option.objects.create(name='opt1')
+    option_2 = Option.objects.create(name='opt2')
+    option_collection_1 = OptionCollection.objects.create(
+        option_collection_hash='my_option_hash_1',
+        option=option_1)
+    option_collection_2 = OptionCollection.objects.create(
+        option_collection_hash='my_option_hash_2',
+        option=option_2)
+    platform_1 = MachinePlatform.objects.create(
+        os_name='win1',
+        platform='win8',
+        architecture='x86')
+    platform_2 = MachinePlatform.objects.create(
+        os_name='win2',
+        platform='win7',
+        architecture='x86')
+    signature1 = PerformanceSignature.objects.create(
+        repository=repo_ref('default', "dev1", "test_treeherder_1"),
+        signature_hash=(30*'t'),
+        framework=test_perf_framework,
+        platform=platform_1,
+        option_collection=option_collection_1,
+        suite='mysuite_1',
+        test='mytest_1',
+        has_subtests=False,
+        last_updated=datetime.datetime.now() + datetime.timedelta(hours=1)
+    )
+    signature2 = PerformanceSignature.objects.create(
+        repository=repo_ref('default', "dev2", "test_treeherder_2"),
+        signature_hash=(40*'t'),
+        framework=test_perf_framework,
+        platform=platform_2,
+        option_collection=option_collection_2,
+        suite='mysuite_2',
+        test='mytest_2',
+        has_subtests=False,
+        last_updated=datetime.datetime.now()
+    )
```

Comment:
I think you should be able to get away without most of this. You should be able to just use `test_repository` for the repository for all signatures. For the first signature, just use `test_perf_signature`. For the second, you can inherit most of its properties, just rename the suite and test. E.g.:


```py
signature2 = PerformanceSignature.objects.create(
  repository=test_repository,
  signature_hash=(40*'u'),
  framework=test_perf_signature.framework,
  platform = test_perf_signature.platfomr,
  option_collection=test_perf_signature.platform
  suite='mysuite_2',
  test='mytest_2',
  has_subtests=False,
  last_updated=datetime.datetime.now())
```